tests/sentry/mail/test_adapter.py

```diff
@@ -279,6 +282,92 @@ def test_simple_notification(self):
         assert msg.subject == "[Sentry] BAR-1 - Hello world"
         assert "my rule" in msg.alternatives[0][0]
 
+    def test_simple_notification_generic(self):
+        """Test that an issue that is neither error nor performance type renders a generic email template"""
+        event = self.store_event(
+            data={"message": "Hello world", "level": "error"}, project_id=self.project.id
+        )
+        event = event.for_group(event.groups[0])
+        occurrence = IssueOccurrence(
+            uuid.uuid4().hex,
+            uuid.uuid4().hex,
+            ["some-fingerprint"],
+            "something bad happened",
+            "it was bad",
+            "1234",
+            {"Test": 123},
+            [
+                IssueEvidence("Evidence 1", "Value 1", True),
+                IssueEvidence("Evidence 2", "Value 2", False),
+                IssueEvidence("Evidence 3", "Value 3", False),
+            ],
+            GroupType.PROFILE_BLOCKED_THREAD,
+            ensure_aware(datetime.now()),
+        )
+        occurrence.save()
+        event.occurrence = occurrence
+
+        event.group.type = GroupType.PROFILE_BLOCKED_THREAD
+
+        rule = Rule.objects.create(project=self.project, label="my rule")
+        ProjectOwnership.objects.create(project_id=self.project.id, fallthrough=True)
+
+        notification = Notification(event=event, rule=rule)
+
+        with self.options({"system.url-prefix": "http://example.com"}), self.tasks():
+            self.adapter.notify(notification, ActionTargetType.ISSUE_OWNERS)
+
+        msg = mail.outbox[0]
+        assert msg.subject == "[Sentry] BAR-1 - Hello world"
```

Comment:
I think that generally we might want to use `occurrence.issue_title` if available. Or I guess `<short_id> - <issue_title>`. Maybe fine as a followup though.

Reply:
This is actually the same as we do for error and performance issue titles, should it be different?

Reply:
Nvm I see what you're saying ... looking into how we generate the subject.

Reply:
For performance issues we do `<short_id> - <issue_type>`, do we want to do the same here?